Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py

```diff
@@ -259,7 +259,14 @@ def create_pull_request(cls, repository, args, branch_point):
             sys.stderr.write("'{}' is not a remote in this repository\n".format(source_remote))
             return 1
 
-        rebasing = args.rebase or (args.rebase is None and repository.config().get('pull.rebase'))
+        if args.rebase is not None:
+            rebasing = args.rebase or (args.rebase is None and repository.config().get('pull.rebase'))
```

Comment:
`(args.rebase is None and repository.config().get('pull.rebase'))` will always be false as this is in `args.rebase is not None:` condition.
So we can just do 
```
 rebasing = args.rebase is not None or repository.config().get(
                 'webkitscmpy.auto-rebase-branch',
                 repository.config().get('pull.rebase', 'true'),
             ) == 'true
```